share/completions/tree.fish

```diff
@@ -26,33 +29,40 @@ complete -c tree -s g -d 'Displays file group owner or GID number'
 complete -c tree -s s -d 'Print the size in bytes of each file'
 complete -c tree -s h -d 'Print the size in a more human readable way'
 complete -c tree -l si -d 'Like -h, but use in SI units (powers of 1000)'
+complete -c tree -l du -d 'Compute size of directories by their contents'
 complete -c tree -s D -d 'Print the date of last modification or (-c) status change'
+complete -c tree -l timefmt -x -d 'Print and format time according to the format <f>'
 complete -c tree -s F -d 'Appends \'/\', \'=\', \'*\', \'@\', \'|\' or \'>\' as per ls -F'
 complete -c tree -l inodes -d 'Print inode number of each file'
 complete -c tree -l device -d 'Print device ID number to which each file belongs'
 
-
 ## Sorting options
 complete -c tree -s v -d 'Sort files alphanumerically by version'
-complete -c tree -s r -d 'Sort files in reverse alphanumeric order'
 complete -c tree -s t -d 'Sort files by last modification time'
 complete -c tree -s c -d 'Sort files by last status change time'
 complete -c tree -s U -d 'Leave files unsorted'
+complete -c tree -s r -d 'Reverse the order of the sort'
 complete -c tree -l dirsfirst -d 'List directories before files (-U disables)'
+complete -c tree -l filesfirst -d 'List files before directories (-U disables)'
+complete -c tree -l sort -d 'Select sort' -xa 'name version size mtime ctime'
 
 ## Graphics options
 complete -c tree -s i -d 'Don\'t print indentation lines'
 complete -c tree -s A -d 'Print ANSI lines graphic indentation lines'
-complete -c tree -s S -d 'Print with ASCII graphics indentation lines'
+complete -c tree -s S -d 'Print with CP437 (console) graphics indentation lines'
```

Comment:
I feel like "ASCII graphics" is going to be more comprehensible than "CP437 (console) graphics"?

Reply:
ASCII contains only 128 values and no pseudo-graphics at all, while CP437  _does_ contain pseudo-graphics — which is enabled by this option. So, CP437 feels to be the correct name here. 

Also, this is what we got in [upstream](https://gitlab.com/OldManProgrammer/unix-tree/-/blob/c70f1b9adbe98016b658e25b05978905f3f85d92/tree.c#L647) right now (the corresponding [change](https://gitlab.com/OldManProgrammer/unix-tree/-/commit/be2d80bd8b64064b4050fd3d8432c762bae09f25#94d3c813985dd997752870f2f64fb02f61c2eaa3_563_654) in tree was introduced more than eight years ago).